todo.js

```diff
@@ -0,0 +1,112 @@
+import { v4 as uuidv4 } from 'uuid'
+
+export class Todo {
+    #id
+    #isDone
+    #text
+
+    constructor(text) {
+        this.#id = uuidv4()
+        this.#isDone = false
+        this.#text = text
+    }
+
+    getId() {
+        return this.#id
+    }
+
+    getIsDone() {
+        return this.#isDone
+    }
+
+    setIsDone(value) {
```

Comment:
I would change it to `toggleIsDone`, you can simply change value to the opposite:
```
toggleIsDone() {
    this.#isDone = !this.#isDone
}
```